<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUsrDlg</class>
 <widget class="QDialog" name="NewUsrDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>711</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新增用户</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="NamelineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="PwdlineEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd2lineEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码确认：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>权限：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="authcomboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>访客</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>管理员</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="TelephonelineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>电话：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmaillineEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱地址：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="RemarkstextEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="Titlelabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>新增用户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>NewUsrDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
